chromium_src/base/json/json_writer.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_BASE_JSON_JSON_WRITER_H_
+#define BRAVE_CHROMIUM_SRC_BASE_JSON_JSON_WRITER_H_
+
+// Adding a value to allow us to serialise 64-bit from blobs.
+#define OPTIONS_PRETTY_PRINT \
+  OPTIONS_SERIALISE_64BIT_NUMBERS = 1 << 3, OPTIONS_PRETTY_PRINT
+#define pretty_print_ \
```

Comment:
Can we guard somehow that that value(`1 << 3`) is not reuesed by chromium in future without us knwoning about it?